be/src/exec/pipeline/aggregate/sorted_aggregate_streaming_sink_operator.h

```diff
@@ -0,0 +1,50 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Inc.
+
+#pragma once
+
+#include <memory>
+
+#include "exec/pipeline/operator.h"
+#include "exec/vectorized/aggregator.h"
+
+namespace starrocks::pipeline {
+// TODO: implements cache-relation method
+class SortedAggregateStreamingSinkOperator : public Operator {
+public:
+    SortedAggregateStreamingSinkOperator(OperatorFactory* factory, int32_t id, int32_t plan_node_id,
+                                         int32_t driver_sequence,
+                                         std::shared_ptr<SortedStreamingAggregator> aggregator);
+    ~SortedAggregateStreamingSinkOperator() override = default;
+
+    bool has_output() const override { return false; }
+    bool need_input() const override { return !is_finished(); }
+    bool is_finished() const override;
+    Status set_finishing(RuntimeState* state) override;
+
+    Status prepare(RuntimeState* state) override;
+    void close(RuntimeState* state) override;
+
+    StatusOr<vectorized::ChunkPtr> pull_chunk(RuntimeState* state) override;
+    Status push_chunk(RuntimeState* state, const vectorized::ChunkPtr& chunk) override;
+
+private:
+    bool _is_finished = false;
+    std::shared_ptr<SortedStreamingAggregator> _aggregator;
+};
+
+class SortedAggregateStreamingSinkOperatorFactory final : public OperatorFactory {
+public:
+    SortedAggregateStreamingSinkOperatorFactory(int32_t id, int32_t plan_node_id,
+                                                StreamingAggregatorFactoryPtr aggregator_factory)
+            : OperatorFactory(id, "aggregate_streaming_sink", plan_node_id),
```

Comment:
`aggregate_streaming_sink` -> `sorted_aggregate_streaming_sink`

Reply:
we can do this later. we also need do some change for visual profile